test/redirect_server_example.js

```diff
@@ -2,15 +2,16 @@
 
 require('decache')('../example/server.js'); // ensure we have a fresh module
 var server = require('../example/server.js');
-var Hoek = require('hoek');
+var Hoek = require('@hapi/hoek');
 
 var config = {
   "401": { // if the statusCode is 401 redirect to /login page/endpoint
     "redirect": "/login"
   },
   "403": {
     "redirect": function (request) {
-      return "/login?redirect=" + request.url.path
+      var redirectString = request.url.pathname + request.url.search;
```

Comment:
just wanted to do this 👍

Reply:
I just copied over the code you wrote in #70 to make the test/example server consistent.
(_the test was failing on my `localhost` when I was trying to update the dependencies ..._)